airqo-platform/AirQo-frontend
# [WEBSITE] Analytics Page

#### Summary of Changes (What does this PR do?)

- Analytics page implementation
- Impact numbers correction
- Events page styling fixes
- Products links added to footer

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?

- View analytics page in deploy preview
- Toggle to see responsiveness

#### What are the relevant tickets?

- [WEB-380]

#### Screenshots
![image](https://user-images.githubusercontent.com/53012890/228805172-d75301d4-9550-474d-8f3c-6aff7f95bc98.png)

[WEB-380]: https://airqoteam.atlassian.net/browse/WEB-380?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
